spec/controllers/account_reset/delete_account_controller_spec.rb

```diff
@@ -100,6 +104,94 @@
         t('errors.account_reset.granted_token_expired', app_name: APP_NAME),
       )
     end
+
+    it 'logs info about user verified account' do
+      user = create(:user, :proofed, confirmed_at: Time.zone.now.round)
+      create_account_reset_request_for(user)
+      grant_request(user)
+      session[:granted_token] = AccountResetRequest.first.granted_token
+
+      delete :delete
+
+      expect(@analytics).to have_logged_event(
+        'Account Reset: delete',
+        user_id: user.uuid,
+        success: true,
+        errors: {},
+        mfa_method_counts: { phone: 1 },
+        identity_verified: true,
+        account_age_in_days: 0,
+        account_confirmed_at: user.confirmed_at,
+      )
+      expect(response).to redirect_to account_reset_confirm_delete_account_url
+    end
+
+    it 'logs info about user biometrically verified account' do
+      user = create(:user, :proofed_with_selfie, :with_phone)
+      create_account_reset_request_for(user)
+      grant_request(user)
+      session[:granted_token] = AccountResetRequest.first.granted_token
+
+      delete :delete
+
+      expect(@analytics).to have_logged_event(
+        'Account Reset: delete',
+        user_id: user.uuid,
+        success: true,
+        errors: {},
+        mfa_method_counts: { phone: 1 },
+        identity_verified: true,
+        account_age_in_days: 0,
+        account_confirmed_at: user.confirmed_at,
+      )
+      expect(response).to redirect_to account_reset_confirm_delete_account_url
+    end
+
+    it 'logs info about user with a verified by mail account' do
+      user = create(:user, :proofed_with_gpo)
+      create_account_reset_request_for(user)
+      grant_request(user)
+      session[:granted_token] = AccountResetRequest.first.granted_token
+
+      delete :delete
+
+      expect(@analytics).to have_logged_event(
+        'Account Reset: delete',
+        user_id: user.uuid,
+        success: true,
+        errors: {},
+        mfa_method_counts: { phone: 1 },
+        identity_verified: true,
+        account_age_in_days: 0,
+        account_confirmed_at: user.confirmed_at,
+      )
+      expect(response).to redirect_to account_reset_confirm_delete_account_url
+    end
+
+    it 'logs info about user pending in person verification account' do
+      user = create(
+        :user,
+        :proofed_in_person_enrollment,
+        :with_phone,
+      )
+      create_account_reset_request_for(user)
+      grant_request(user)
+      session[:granted_token] = AccountResetRequest.first.granted_token
+
+      delete :delete
+
+      expect(@analytics).to have_logged_event(
+        'Account Reset: delete',
+        user_id: user.uuid,
+        success: true,
+        errors: {},
+        mfa_method_counts: { phone: 1 },
+        identity_verified: true,
+        account_age_in_days: 0,
+        account_confirmed_at: user.confirmed_at,
+      )
```

Comment:
Neither of these log assertions give us anything to distinguish IPP or GPO? I think this is likely a problem with the factories not creating the expected proofing components. Want to take a stab at that?

Also, mentioned in an earlier review comment, we shouldn't bother asserting redirect when we're concerned with the log output.

Reply:
Yeah I'll try to get those added to the factories.